src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java

```diff
@@ -499,4 +499,28 @@
     )
     @ConfigEditorBoolean
     public boolean dwarvenTextures = false;
+    @Expose
+    @ConfigOption(
+            name = "Crystal Hollows Textures",
+            desc = "Allows texture packs to retexture blocks in the Crystal Hollows. If you don't have a texturepack that does this, you should leave this off"

```

Comment:
```suggestion
            desc = "Allows texture packs to retexture blocks in the Crystal Hollows. If you don't have a texture pack that does this, you should leave this off"
```
you put a space between the other texture packs